templates/compose/usesend-with-smtp.yaml

```
@@ -0,0 +1,91 @@
+# documentation: https://docs.usesend.com/self-hosting/overview#step-5%3A-smtp-proxy-server-optional
+# slogan: Usesend is an open-source alternative to Resend, Sendgrid, Mailgun and Postmark etc.
+# category: messaging
+# tags: resend, mailer, marketing emails, transaction emails, self-hosting, postmark
+# logo: svgs/usesend.svg
+# port: 3000
+
+services:
+  postgres:
+    image: postgres:16
+    environment:
+      - POSTGRES_USER=${SERVICE_USER_POSTGRES}
+      - POSTGRES_PASSWORD=${SERVICE_PASSWORD_POSTGRES}
+      - POSTGRES_DB=${SERVICE_DB_POSTGRES:-usesend}
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}"]
+      interval: 5s
+      timeout: 20s
+      retries: 10
+    volumes:
+      - usesend-postgres-data:/var/lib/postgresql/data
+
+  redis:
+    image: redis:7
+    volumes:
+      - usesend-redis-data:/data
+    command: ["redis-server", "--maxmemory-policy", "noeviction"]
+    healthcheck:
+      test:
+        - CMD
+        - redis-cli
+        - PING
+      interval: 5s
+      timeout: 10s
+      retries: 20
+
+  usesend-proxy:
+    image: "usesend/smtp-proxy:v1.7.1"
+    user: "0"
+    environment: SMTP_AUTH_USERNAME=${SMTP_USERNAME:-usesend}
+      USESEND_BASE_URL=${SERVICE_FQDN_USESEND}
+      UNSEND_API_KEY_PATH=/data/certs/key.pem
+      UNSEND_API_CERT_PATH=/data/certs/cert.pem
+    ports:
+      - "25:25" # Standard SMTP
+      - "587:587" # SMTP with STARTTLS
+      - "2587:2587" # Alternative SMTP port
+      - "465:465" # SMTP over SSL/TLS
+      - "2465:2465" # Alternative SMTP port
+    volumes:
+      - '/data/coolify/certs/$SERVICE_FQDN_UNSEND_PROXY/key.pem:/data/certs/key.pem:ro'
+      - '/data/coolify/certs/$SERVICE_FQDN_UNSEND_PROXY/cert.pem:/data/certs/cert.pem:ro'
+    depends_on:
+      - usesend
+
+  usesend:
+    image: usesend/usesend:v1.7.1
+    expose:
+      - 3000
+    environment:
+      - SERVICE_URL_USESEND_3000
+      - DATABASE_URL=postgresql://${SERVICE_USER_POSTGRES}:${SERVICE_PASSWORD_POSTGRES}@postgres:5432/${SERVICE_DB_POSTGRES:-usesend}
+      - NEXTAUTH_URL=${SERVICE_URL_USESEND}
+      - NEXTAUTH_SECRET=${SERVICE_BASE64_64_NEXTAUTHSECRET}
+      - AWS_ACCESS_KEY=${AWS_ACCESS_KEY:?}
+      - AWS_SECRET_KEY=${AWS_SECRET_KEY:?}
+      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:?}
+      - GITHUB_ID=${GITHUB_ID:?}
+      - GITHUB_SECRET=${GITHUB_SECRET:?}
+      - REDIS_URL=redis://redis:6379
+      - NEXT_PUBLIC_IS_CLOUD=${NEXT_PUBLIC_IS_CLOUD:-false}
+      - API_RATE_LIMIT=${API_RATE_LIMIT:-1}
+      - HOSTNAME=0.0.0.0
+    depends_on:
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD-SHELL", "wget -qO- http://usesend:3000 || exit 1"]
+      interval: 5s
+      retries: 10
+      timeout: 2s
+
+  traefik-certs-dumper:
+    image: ghcr.io/kereis/traefik-certs-dumper:1.8.12
+    environment: DOMAIN=${SERVICE_FQDN_UNSEND_PROXY}
```

The `SERVICE_FQDN_UNSEND_PROXY` will have value of random subdomain and user cannot change this since you don't have a `SERVICE_URL_UNSEND_PROXY` and proxy service doesn't have a public url so which domain is needed here?

Also follow same formatting for env:

```suggestion
    environment: 
      - DOMAIN=${SERVICE_FQDN_UNSEND_PROXY}
```